<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>731</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 68, 121);
font: 36pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Sistema Automático de Medida</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 68, 121);
font: 18pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Selección de instrumentos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 68, 121);
font: 18pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Osciloscopio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 68, 121);
font: 18pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Generador:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="osciloscopio">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 68, 121);
font: 15pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Juan Ignacio Ochotorena Pérez-Argos </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 15pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 15pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Volver a buscar instrumentos</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777205</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QMenu" name="menudfgagf">
    <property name="title">
     <string>Sistema Automatico de Medida</string>
    </property>
   </widget>
   <addaction name="menudfgagf"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
